### ISSUE TITLE
Multiple-Step Forecasting - Predict one series using multiple series as predictors

### ISSUE THREAD

**Issue by AVPokrovsky** [JoaquinAmatRodrigo/skforecast#776]
Hi, in you code examples your always include predicted variable ("o3" below) in the the input series. I have some issue to add predicted variable into series and would like to exclude it from series. Is it possible?

# Model creation
# ==============================================================================
# Time series used in the training. Now, it is multiseries
series = ['pm2.5', 'co', 'no', 'no2', 'pm10', 'nox', 'o3', 'veloc.', 'direc.','so2'] 
levels = ["o3"]

**Comment by FernandoCarazoMelo** [comment]
Hi @AVPokrovsky !

Do you mean the ForecarterRnn?

If that's the case, yes- it's possible to exclude the target series from the predictor ones. The reason why we included it is that it's usually the most predictive feature.

**Comment by AVPokrovsky** [comment]
Yes, it's rnn. It seems, when I tried to delete it from series got an error, something that no levels variable...

**Comment by FernandoCarazoMelo** [comment]
Coulds you share the error?

**Comment by AVPokrovsky** [comment]
Hi, error:
ValueError: `levels` defined when initializing the forecaster must be included in `series` used for trainng. {'adjclose'} not found.

**Comment by AVPokrovsky** [comment]
levels='adjclose'
series
              high   close
date                      
2022-01-17  6734.0  7115.0
2022-01-18  6587.0  6849.0
2022-01-19  6670.0  7093.0
2022-01-20  6683.0  7029.0
2022-01-21  6514.0  7019.0
           ...     ...
2023-05-27  5298.5  6432.5
2023-05-28  5298.5  6432.5
2023-05-29  5547.5  6674.5
2023-05-30  5698.5  6666.0
2023-05-31  5594.0  6705.0

**Comment by JoaquinAmatRodrigo** [comment]
Hi everyone! 👋

It’s been a while since the last activity on this issue, so I’m going to close it to help us keep our backlog manageable and focused on active tasks.

If you are still experiencing this problem or believe this feature is still highly relevant for the latest version, please feel free to leave a comment or reopen the issue. Thank you for your contribution!
